### DISCUSSION TITLE
Welcome to competiitve_coding Discussions!

### DISCUSSION THREAD

**Discussion by warriorwizard** [warriorwizard/competiitve_coding#12] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amit-0001** [comment, marked as answer]
i was wondering is there a way to gain something that is important or essential

**Comment by warriorwizard** [comment]
yeah sure there will be just explore different thing for now
